project/src/World/DockingPoint/DockingPoint.tscn

```diff
@@ -31,6 +31,11 @@ texture = ExtResource( 2 )
 [node name="Sprite" type="Sprite" parent="."]
 scale = Vector2( 0.703517, 0.703517 )
 
+[node name="PlayerRotationRig" type="Node2D" parent="Sprite"]
```

Comment:
Don't forget the remove the old remote transform rig from the dockingpoint scene.

Reply:
a0ee1a3393e29ac613ebcbfce61773b560e0801c